public/content/en/basics/delegates.md

```diff
@@ -12,6 +14,8 @@ A function can also be a parameter to another function:
 
 `doer` can then be called like any other normal function.
 
+### Local functions with contexts
```

Comment:
I wasn't sure whether context or context__s__ is correct.
My argumentation for context is that a function can only have one and it's referred to a single function.